cms/static/js/factories/register.js

```diff
@@ -29,5 +29,30 @@ define(['jquery', 'jquery.cookie'], function($) {
                 }
             });
         });
+
+        $('input#password').blur(function() {
+            // Uninitialize the errors on blur
+            var $formErrors = $('#password_error');
+            $formErrors.empty();
+            $formErrors.addClass('hidden');
+
+            var data = {
+                password: $('#password').val()
+            };
+            $.ajax({
```

Comment:
$.ajax is edx-deprecated in favor of the JS-native fetch() (which edx-platform has a "whatwg-fetch" transpiler for). You already wrote this, so probably not worth rewriting in fetch. Plus, it's currently only used in es6 code, so you'd have to figure out the es5-babel-side of things. But next time, maybe.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
Good to know! Might be worth us taking up the approach that was done back in the Coffeescript days of if you change an ES5 file, convert it.